assets/locations.xml

```diff
@@ -18,7 +18,17 @@
         icon="swim"
         x="1200"
         y="1000"
-    />
+     />  
+  
+    <location
+        name="Naizatas"
+        mapUrl="https://goo.gl/maps/ipigT1CXaa2YTSJK9"
+        image="images/location_images/Naizatas.jpeg"
+        content="Mount Naizatas with a height of 568 meters above sea level is located in the Bayanaul National Natural Park in the Bayanul district of the Pavlodar region."        
```

Comment:
Since this app is about Bayanaul, you don't need to add where it is located :) But the tourist could find it valuable to know what is special about Naizatas and why you think it is worth a visit. For example, is it because it has a loaf-like shape? or there are not trees growing on it? What does Naizatas mean and why it was named like that? You can add all of that if you wanted to.